pymc/sampling/jax.py

```diff
@@ -67,6 +68,14 @@
 )
 
 
+@jax_funcify.register(IdentityOp)
```

Comment:
PyTensor has an identity Op already that gets automatically removed during compilation (so a dispatch wouldn't be needed). Can we use that instead and remove the PyMC one?

Reply:
yes, would be much better